OracleTuxedo/OracleTSAM/README.md

```diff
@@ -0,0 +1,166 @@
+Oracle TSAM Plus on Docker
+===============
+Sample Docker build files to facilitate installation, configuration, and environment setup for DevOps users. This image has also been certified with [Oracle Container Cloud Service (OCCS)](https://cloud.oracle.com/container). For more information about **Oracle Tuxedo System and Applications Monitor Plus (TSAM Plus)** please see the [Oracle TSAM Plus Online Documentation](http://docs.oracle.com/cd/E72452_01/tsam/docs1222/index.html).
+
+This project offers Dockerfile for building:
+ * Oracle TSAM Plus 12c Release 2 (12.2.2) + Rolling Patch 003
+
+## Dependencies
+This project depends on the [Oracle Server JRE 8 Docker Image](../..//OracleJava). So, before you proceed to build this image, make sure the image `oracle/serverjre:8` has been built locally or is accessible in a remote Docker registry.
+
+**IMPORTANT:** You will have to provide the required installation binaries and put them into the `dockerfiles/<version>` folder.
+
+For this image following installation media / binaries are required:
```

Comment:
There is a description "Please refer to the **REQUIRED FILES TO BUILD THIS IMAGE** section in the [Dockerfile](dockerfiles/12.2.2/Dockerfile) for more detail." right after those lines. All the required information are in the `Dockerfile`:

```
# REQUIRED FILES TO BUILD THIS IMAGE
# ----------------------------------
# (1) Oracle Tuxedo System and Applications Monitor (TSAM) Plus 12cR2 (12.2.2) GA Installer: tsam122200_64_Linux_x86.zip
#     Download from http://www.oracle.com/technetwork/middleware/tuxedo/downloads/index.html
#
# (2) Oracle TSAM Plus 12.2.2 Rolling Patch 003: p25448043_12220_Linux-x86-64.zip
#     Download from https://updates.oracle.com/Orion/Services/download/p25448043_12220_Linux-x86-64.zip?aru=20968945&patch_file=p25448043_12220_Linux-x86-64.zip
#
# (3) Oracle Database Instant Client: instantclient-basiclite-linux.x64-12.2.0.1.0.zip & instantclient-sqlplus-linux.x64-12.2.0.1.0.zip
#     Download from http://www.oracle.com/technetwork/topics/linuxx86-64soft-092277.html
```

Does this work?